README.md

```diff
@@ -17,7 +18,7 @@ import Foo.Handler.GoodbyeWorld
 
 ### Output:
 ```haskell
-{-# LANGUAGE PartialTypeSignatures #-}
+{-# LANGUAGE PartialTypeSignatures -freduction-depth=0 #-}
```

Comment:
Can you put a ghc option in a language pragma? No right?

Reply:
Oops, you're right this was just a readme mistake